packages/rocketchat-ui/client/views/app/directory.js

```diff
@@ -31,6 +22,13 @@ function directorySearch(config, cb) {
 					createdAt: timeAgo(result.createdAt)
 				};
 			}
+			return {
```

Comment:
wrapping this in an `else` would be more elegant

Reply:
handled with else block.